src/panels/lovelace/editor/lovelace-editor/hui-lovelace-editor.ts

```diff
@@ -45,10 +57,36 @@ export class HuiLovelaceEditor extends LitElement {
           .configValue="${"title"}"
           @value-changed="${this._valueChanged}"
         ></paper-input>
+        
+        <ha-formfield
+          .label=${this.hass.localize(
+            "ui.panel.lovelace.editor.edit_lovelace.append_view_title"
+          )}
+          .dir=${computeRTLDirection(this.hass)}
+        >
+          <ha-switch
+            .checked=${this._append_view_title !== false}
```

Comment:
```suggestion
            .checked=${this._append_view_title}
```

Here is where this would be going wrong as well. You are checking to see if its not false. If its undefined we don't want to append either